backends-velox/src/test/scala/io/glutenproject/execution/VeloxLiteralSuite.scala

```diff
@@ -126,12 +132,6 @@ class VeloxLiteralSuite extends VeloxWholeStageTransformerSuite {
   }
 
   test("Literal Fallback") {
-    validateFallbackResult("SELECT array()")
-    validateFallbackResult("SELECT array(array())")
-    validateFallbackResult("SELECT array(map())")
-    validateFallbackResult("SELECT map()")
-    validateFallbackResult("SELECT map(1, array())")
-    validateFallbackResult("SELECT map(1, map())")
     validateFallbackResult("SELECT array(null)")
```

Comment:
Do you know why does it fallback ?

Reply:
yes, it's not related to empty literal, but NullType support.
I'll fix it in another PR, otherwise this PR is too big.